tests/test_hear_keyboard_interrupt.py

```
@@ -84,14 +90,20 @@ def test_hear_keyboard_interrupt(self):
 				text=True
 			)
 			try:
-				time.sleep(1)  # Allow server to start
+				time.sleep(STARTUP_DELAY_SECONDS)  # Allow server to start
```

Ruff (0.8.2)

Undefined name `STARTUP_DELAY_SECONDS`

(F821)